dnsmasq.template

```diff
@@ -0,0 +1,25 @@
+# Template file for dnsmasq generation
+# dnsmasq will default to sending v4 + v6 address of dnsmasq server
+
+# Enforce some standards
+bogus-priv
```

Comment:
if these are in the template sample, we probably want a comment to explain what they are

Reply:
Will do.